weblab/experiments/models.py

```diff
@@ -66,12 +101,18 @@ class ExperimentVersion(UserCreatedModelMixin, VisibilityModelMixin, models.Mode
     )
     return_text = models.TextField(blank=True)
     task_id = models.CharField(max_length=50, blank=True)
-    model_version = models.CharField(max_length=50)
-    protocol_version = models.CharField(max_length=50)
 
     def __str__(self):
         return '%s at %s: (%s)' % (self.experiment, self.created_at, self.status)
 
+    @property
+    def name(self):
+        return str(self.experiment.versions.filter(pk__lte=self.pk).count())
```

Comment:
Comparing the `pk` worries me a little - can it always be relied on that older pks are lower? Using `created_at` might be more robust.

Reply:
I was assuming that `pk` would be an auto-increment field so this would be safe. But using `created_at` works too!